mods/d2k/maps/worm-graveyard/destroyable-tiles.yaml

```
@@ -0,0 +1,198 @@
+^new_tile_undestroyable:
+	Inherits@1: ^ExistsInWorld
+	Inherits@2: ^SpriteActor
+	FrozenUnderFog:
+	WithSpriteBody:
+	CombatDebugOverlay:
+	Building:
+		TerrainTypes: Rock, Concrete, Cliff, Sand, Rock, Transition, Spice, SpiceSand, Dune,
+	RenderSprites:
+		Palette: terrain
+	Health:
+		HP: 20000
+	Armor:
+		Type: invulnerable
+	HitShape:
+		UseTargetableCellsOffsets: false
+		Type: Rectangle
+			TopLeft: -512, -512
+			BottomRight: 512, 512
+	MapEditorData:
+		Categories: System
+	QuantizeFacingsFromSequence:
+	Interactable:
+	Tooltip:
+		GenericName: pasage
+		Name: passage
+		ShowOwnerRow: false
+
+^new_tile_destroyable:
+	Inherits: ^new_tile_undestroyable
+	Targetable:
+		TargetTypes: Ground, Structure
+		RequiresForceFire: true
+	Armor:
+		Type: wall
+	Health:
+		HP: 30000
+	Tooltip:
+		GenericName: pasage (Destroable)
+		Name: passage (Destroable)
```

```suggestion
		Name: Passage (destroyable)
```

these should be using our translations system, and do standardise the capital letters and fix spelling

---

Do we have some dokumentation on how translations works? Can I just overide en.ftl in  custom maps?

---

you can run the `--extract-yaml-strings` command. Then you can format it to match the others

---

the system is currently in progress, and no, maps can't overwrite mod level translations. You need unique translation keys. The lint checks, that it `--check-yaml` should catch any errors you might have. Though the fact that this PR didn't upset the CI gods makes me suspicious that something broke